.travis.yml

```diff
@@ -62,7 +62,6 @@ before_install:
 install:
   - env
   - pip install -r requirements/test.txt
-  - pip install coveralls
   - pip install mock
   - python setup.py bdist_egg
   - if [ "$JYTHON" == "true" ]; then
```

Comment:
There's a "python -m pip install coveralls" in this if block, should there be?

Reply:
Yes - it installs coveralls for _Python_ when ```$JYTHON == "true"``` (pip install coveralls is also in the "else" block)